vignettes/algorithm.Rmd

```diff
@@ -218,7 +218,7 @@ and we can thereby calculate the left and right hand sides for the WLS estimatin
 equation. We solve this equation with a robust Cholesky decomposition with pivoting.
 The advantage is that we can reuse this decomposition for calculating the
 covariance matrix of $\hat{\beta}$, i.e. $K = (X^\top W X)^{-1}$, by supplying the
-identity matrix as alternative right hand side.
+identity matrix as alternative right hand side. When weights $Z_i$ is specified, we replace $L_i$ with $Z_i^{-1/2} L_i$ in the above calculation.
```

Comment:
Also here I don't think we need this since in line 195 we refer to $\Sigma_i$
```suggestion
identity matrix as alternative right hand side.
```

Reply:
We could instead describe additional details how $L_i$ is calculated but not sure if needed?